SSW.Consulting.Application/Interfaces/IStorageProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+
+namespace SSW.Consulting.Application.Interfaces
+{
+	public interface IStorageProvider
+	{
+		Task UploadBlob(string containerName, string filename, byte[] contents);
```

Comment:
just realized we should probably add `Streams` for uploading/downloading too. Might help for large files.